docs/src/main/sphinx/admin/properties.rst

```diff
@@ -21,3 +21,80 @@ may be used to tune Trino or alter its behavior when required.
     Logging <properties-logging>
     Web UI <properties-web-interface>
     Regular expression function <properties-regexp-function>
+
+Property value types
+--------------------
+
+Configuration properties in Trino take values of different types, some standard
+and some non-standard. Supported values may be further limited on a per-property
+basis.
+
+.. _prop-type-boolean:
+
+``boolean``
+^^^^^^^^^^^
+
+Properties of a ``boolean`` type support two values, ``true`` or ``false``
+in lowercase.
```

Comment:
nit: using words like MUST, SHOULD, COULD help enforce things IMO. WDYT?

```suggestion
Properties of a ``boolean`` type support two values, ``true`` or ``false``.
The values must be lowercase.
```

Reply:
I like that

Reply:
Actually they don't have to be lowercase.
https://github.com/airlift/airlift/blob/3546f4b12884828372d30a8ae821327ad87441e8/configuration/src/main/java/io/airlift/configuration/ConfigurationFactory.java#L573-L577

Reply:
Added "must" and similar where appropriate. Removed the lowercase requirement.